native/python/zjb_py.cpp

```diff
@@ -0,0 +1,209 @@
+#include "zjb_py.hpp"
+#include <iostream>
+
+static void e2a_inplace(std::string &s)
```

Comment:
Please refer to my earlier comment about generalizing/placing this in a common area - this applies in the 3 source (.cpp) files in the `python` folder